.circleci/config.yml

```diff
@@ -324,89 +195,90 @@ jobs:
             echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
             ' >> ~/.ssh/known_hosts
       - run:
-          name: Notify of new release
+          name: Update Changelog and notify
           command: |
             version=$(get-new-version.sh)
             changelog.sh "$version"
+      - slack/notify:
+          channel: C014UMRC4AJ
+          mentions: here
+          color: '#78c043'
+          message: "🌍 A new Planet 4 release is currently being deployed.\nCheck the <https://support.greenpeace.org/planet4/tech/changelog|Changelog> for the full list of changes"
```

Comment:
Yes, that's indeed a nice idea. It would require some coordination with NROs first, because the `channel` field would have to be different for each NRO and we won't necessarily have access to each NRO Slack workspace to setup the integration.